compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/numeric-literal-as-object-property-key.js

```diff
@@ -0,0 +1,6 @@
+function Test() {
+  const obj = {
+    21: 'dimaMachina'
+  }
+  return <div>{obj[21]}</div>
+}
```

Comment:
please add a fixture here so that we can test evaluation with/without compilation:

```
export const FIXTURE_ENTRYPOINT = {
  fn: Test,
  params: [{}],
};
```

Reply:
also be sure to update fixtures after this

Reply:
Thank you! Updated! Could you also review my other contribution to the react compiler? https://github.com/facebook/react/pull/31792

I just pushed there similar change 🙂